angularjs-portal-home/src/main/webapp/my-app/search/controllers.js

```diff
@@ -84,21 +86,25 @@ define(['angular', 'portal/search/controllers', 'my-app/marketplace/controllers'
           $('#myuw-results').show();
           $('#wisc-directory-results').show();
           $('#wisc-edu-results').show();
+          $scope.activeFilter = "all";
```

Comment:
Doing this makes the `jquery` implementation redundant. I suggest choosing one and go with it.